app/MyApplicationLoader.scala

```diff
@@ -48,23 +48,25 @@ class MyComponents(context: Context)
 
   val httpClient = com.gu.okhttp.RequestRunners.futureRunner
 
+  implicit val cc = controllerComponents
+
   lazy val router: Routes = new Routes(
     httpErrorHandler,
-    new CachedAssets(assets),
-    new Homepage(commonActions),
+    new CachedAssets(assets, controllerComponents),
+    new Homepage(commonActions, controllerComponents),
     new Management(actorSystem = actorSystem, touchpointBackends, oAuthActions),
```

Comment:
I wasn't sure whether it should be implicit or passed in explicitly, it happened this way because I used search and replace to refactor the class definition.  I'm happy to go one way or the other, or sit on the fence!